siliconcompiler/core.py

```diff
@@ -793,7 +793,7 @@ def get(self, *keypath, field='value', job=None, cfg=None):
             else:
                 cfg = self.cfg
 
-        keypathstr = ','.join(keypath)
+        keypathstr = f'{keypath}'
 
         self.logger.debug(f"Reading from [{keypathstr}]. Field = '{field}'")
```

Comment:
I think we should get rid of the "[ ]" around `{keypathstr}` now, since casting the keypath to a string will include those (I think the keypath would be printed with two brackets now). Same applies to the debug logs in the other related functions.

Reply:
You're right - the f-string formatting uses `()` instead of `[]`, but that should work fine for logging statements.